pillar.example

```diff
@@ -27,9 +27,19 @@ samba_sections:
     force group: sharegroup
     create mask: 0660
     directory mask: 2770
+  user1share:
```

Comment:
This is a good addition. I would like to see this modified so it's all part of the `samba_users` section for adding users, that would also allow for a more simplified loop.